integration-tests/api/__tests__/line-item-adjustments/index.js

```diff
@@ -1,14 +1,15 @@
 const path = require("path")
 const { LineItemAdjustment } = require("@medusajs/medusa")
-const setupServer = require("../../../helpers/setup-server")
-const { useApi } = require("../../../helpers/use-api")
-const { initDb, useDb } = require("../../../helpers/use-db")
+const { useDb } = require("../../../helpers/use-db")
 const cartSeeder = require("../../helpers/cart-seeder")
 const { simpleCartFactory, simpleLineItemFactory } = require("../../factories")
 const {
   simpleDiscountFactory,
 } = require("../../factories/simple-discount-factory")
 
+const startServerWithEnvironment =
```

Comment:
todo: The existing integration tests should not be impacted by the sales channels flag. Could you add some integrations tests and place them in the right sales channels tests suite

Reply:
That way we would introduce a precedent in organising our test suites around feature flags instead of domain. Coincidently sales channel FF corresponds to a domain section but that might not be the case always. I would just want to check with Oli & Seb if that approach is ok.

The more I think about testing with feature flags...as we add new flags, testing could become increasingly complicated, for example, if we add 2 feature flags that impact the same domain sometimes we would have to add taste cases for combinations of flag states - in that case, the current organisation would make more sense, wdyt?

Reply:
I completely agree, and once the feature flag will be removed, we will also have to update the tests, but this has been discussed with seb in order to be sure that the entire domain tests suite is not impacted by the flags

Reply:
Also,
If you look at the sales channels suite in admin and store
Integration, they include cart/product/order tests

Reply:
I ve also already tried to create a sub describe in the domain with another server using the flags but did not work, if you succeed to make that work, then we can move back the other tests to their domains

Reply:
Just chiming in here:

We don't have any set convention for grouping tests although you could infer one as @fPolic is mentioning: grouped by domain. Given that this is not a strong convention but more a coincidence I don't think we have any solid reason to enforce that all tests for a given domain are in the same suite.

For now, I am OK with just placing tests somewhere we can ensure they are run under the correct env (i.e. feature flags). If anyone has better ideas for a convention going forward, I suggest we write an RFC on it and discuss it more broadly later, but this is not a huge priority at this point in my mind.

--
For this particular case, feel free to move the tests wherever works best. Could for example be a new file: line-item-adjustments/ff-sales-channel.ts, or something different.

Reply:
cleaned current test suites and added new ones in `line-item-adjustments/ff-sales-channel.js`

Reply:
Really like that approach 🥰